dev-support/ci/selective_ci_checks.bats

```diff
@@ -121,7 +121,7 @@ load bats-assert/load.bash
   run dev-support/ci/selective_ci_checks.sh 1dd1d0ba3
 
   assert_output -p 'basic-checks=["rat","author","checkstyle","findbugs","unit"]'
-  assert_output -p needs-build=false
+  assert_output -p needs-build=true
```

Comment:
Would the move to source tar ball build increase the unit testing time?

Reply:
Unit testing is not affected, only compile check.  Increase is minimal (the time to create the tarball).  Overall wall-clock time of CI workflow does not increase.